packages/bcer-retailer-app/app/src/components/Sales/withNav.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useContext, useEffect } from 'react';
```

Comment:
Not sure I understand how `withNav` is used, can you elaborate?

Reply:
withNav is a Higher Order Components. You can that every page has "Cancel", "GO Back", "Next" or "Submit". This can make the logic separate from the contents page without having those duplicates button and events.
![image](https://user-images.githubusercontent.com/46575401/135359200-229ac2eb-f8a9-4483-b211-54404e9ef2c4.png)